code/modules/client/preference_setup/loadout/loadout_uniform.dm

```diff
@@ -9,6 +9,18 @@
 	display_name = "cheongsam, white"
 	path = /obj/item/clothing/under/cheongsam
 
+/datum/gear/uniform/cheongsam/red
+	display_name = "cheongsam, red"
+	path = /obj/item/clothing/under/cheongsam/red
+
+/datum/gear/uniform/cheongsam/blue
+	display_name = "cheongsam, blue"
+	path = /obj/item/clothing/under/cheongsam/blue
+
+/datum/gear/uniform/cheongsam/
+	display_name = "cheongsam, black"
+	path = /obj/item/clothing/under/cheongsam/black
```

Comment:
https://github.com/Baystation12/Baystation12/pull/12688 might new useful tho, because it means you can setup a /path without creating the list yourself, just supply the type you want to select from.